python/paddle/v2/framework/framework.py

```diff
@@ -358,6 +363,23 @@ def create_var(self, *args, **kwargs):
             kwargs['initializer'](var, self)
         return var
 
+    def create_persistable_var(self, init_program, initializer, prefix, *args,
+                               **kwargs):
+        """
+        create a persistable var in init_program.global_block and current block.
```

Comment:
Why we need to create the variable in both init_program and main_program. Parameters are created only in init_program.
If init_program and main_program are in the same scope, will the same var_name be a conflict?

Reply:
now framework will create some persistable var or parameter in both init_program and program, and prepend init_op in init_program to initialize the persistable var, when executor create a real variable, if it finds the var is persistable, it will only create it in the root scope of Executor.